gssapi/tests/test_high_level.py

```diff
@@ -551,6 +609,7 @@ def test_copy(self):
     # doesn't actually implement it
 
     @ktu.gssapi_extension_test('rfc6680', 'RFC 6680')
+    @ktu.krb_provider_test(['mit'], 'Heimdal does not implemented for krb5')
```

Comment:
Can you reword (all of) these?  "does not implemented" isn't very clear.  The resulting skip message currently is:

Your GSSAPI (provider heimdal) is known to have problems with Heimdal does not implemented for krb5.